scripts/release_main.js

```diff
@@ -81,8 +97,9 @@ async function init() {
   );
 
   if (answerChangelog === 'no') {
+    // TODO revertCommit() to do this automatically.
     console.log('You may want to revert the changed files!');
-    process.exit();
+    process.exit(1);
   }
 
   await commit({ newVersion });
```

Comment:
dumb q. how do you protect `main` when you're doing this?

Reply:
We don't. It's shitty. Ideally, this should happen after a feature PR is merged:
- GH Action triggers, which opens a new MR with the new version + CHANGELOG.
- You (or any maintainer) review the MR and approve it. Or tweak it if needed, or even close it, in case it doesn't make sense to release a new version.
  - On merge, another action is triggered to do the actual release.

But as I don't know Github Actions I couldn't make it happen :(